src/vxworks/mod.rs

```diff
@@ -253,11 +252,8 @@ s! {
     }
 
     // signal.h
-    pub struct sigaction { // pulled from kernel side,
-        pub sa_u     : ::size_t,
-        // This is a union of two function pointers.
-        // Under the assumption that function pointers are the same
-        // size as other pointers, we can replace the union with size_t
+    pub struct sigaction {
+        pub sa_u     : ::size_t, // actually union of two function pointers
```

Comment:
Why not use an `union` then ?

Reply:
I see on another targets, they are using sighandler_t for similiar case, and sighandler_t is defined as:
unix/mod.rs:30:pub type sighandler_t = ::size_t; 

Can we use the same way on vxWorks? i.e, change size_t to sighandler_t.

Reply:
Those other targets where added when Rust did not had `union`s, so they couldn't use them.